hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/tools/TestDFSAdmin.java

```diff
@@ -1205,9 +1205,9 @@ public void testAllDatanodesReconfig()
     LOG.info("dfsadmin -status -livenodes output:");
     outs.forEach(s -> LOG.info("{}", s));
     assertTrue(outs.get(0).startsWith("Reconfiguring status for node"));
-    assertEquals("SUCCESS: Changed property dfs.datanode.peer.stats.enabled", outs.get(2));
-    assertEquals("\tFrom: \"false\"", outs.get(3));
-    assertEquals("\tTo: \"true\"", outs.get(4));
+    assertEquals("SUCCESS: Changed property dfs.datanode.peer.stats.enabled", outs.get(1));
+    assertEquals("\tFrom: \"false\"", outs.get(2));
+    assertEquals("\tTo: \"true\"", outs.get(3));
```

Comment:
Given that concurrency is at play here, we can do something like this:
```
    assertTrue("SUCCESS: Changed property dfs.datanode.peer.stats.enabled".equals(outs.get(2))
        || "SUCCESS: Changed property dfs.datanode.peer.stats.enabled".equals(outs.get(1)));
```

Reply:
I will modify the code.